backend/src/main/java/ai/verta/modeldb/health/HealthServiceImpl.java

```diff
@@ -105,7 +109,7 @@ void clearStatus(String service) {
       HealthCheckRequest request = HealthCheckRequest.newBuilder().setService(service).build();
       ServingStatus status = getServingStatus(request);
       if (status == null) {
-        LOGGER.trace("Spring custom health check returned with unknown service");
+        LOGGER.error("Spring custom health check returned with unknown service");
         throw new ResponseStatusException(
```

Comment:
```
 public HealthServiceImpl() {
    setStatus("", HealthCheckResponse.ServingStatus.SERVING);
  }
```
was not initialize so `ServingStatus status = getServingStatus(request);` was return the null so from here it was throwing 500.